services/app-api/src/postgres/insertUser.ts

```diff
@@ -23,7 +23,7 @@ export async function insertUser(
                 familyName: user.familyName,
                 email: user.email,
                 role: user.role,
-                stateCode: user.stateCode,
+                stateCode: user.stateCode ?? '',
```

Comment:
And when we pull a CMS user out of the table, we ignore the stateCode field? Is this field required at the Prisma layer?

Reply:
stateCode is an optional in the Prisma schema

Reply:
actually it looks like Prisma uses `null` when it has an optional, fixing this